src/service/repo/signedMessageRepo.ts

```diff
@@ -33,6 +34,7 @@ interface SignedMessageRepoTypes {
     channelId: string,
     message: string,
     channelType: number,
+    attachments: any,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `attachments` parameter is of type `any`. Consider replacing it with a more specific type.
```diff
-    attachments: any,
+    attachments: AttachmentType[],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->